packages/modules/event-bus-local/src/services/event-bus-local.ts

```diff
@@ -35,11 +35,28 @@ export default class LocalEventBusService extends AbstractEventBusModuleService
     // eslint-disable-next-line prefer-rest-params
     super(...arguments)
 
-    this.logger_ = logger
+    this.logger_ = logger ?? console
     this.eventEmitter_ = eventEmitter
     this.groupedEventsMap_ = new Map()
   }
 
+  static logProcessingEvent(
+    eventData: Message,
+    options: Record<string, unknown> = {},
+    totalSubscribers: number,
+    logger: Logger
+  ) {
+    if (
+      totalSubscribers &&
+      !options?.internal &&
+      !eventData.options?.internal
+    ) {
+      logger?.info(
```

Comment:
nit: `?` is not necessary here
```suggestion
      logger.info(
```